game.js

```diff
@@ -33,6 +33,114 @@ monsterImage.onload = function () {
   monsterReady = true;
 };
 monsterImage.src = "images/monster.png";
+// Load the monster image
+var monster1Ready = false;
+var monster1Image = new Image();
+monster1Image.onload = function () {
+  // show the monster image
+  monster1Ready = true;
+};
+monster1Image.src = "images/monster.png";
+// Load the monster image
+var monster2Ready = false;
+var monster2Image = new Image();
+monster2Image.onload = function () {
+  // show the monster image
+  monster2Ready = true;
+};
+monster2Image.src = "images/monster.png";
+// Create the game objects
+var hero = {
+  speed: 256, // movement speed of hero in pixels per second
+  height: 36,
+  width: 31
+};
+var monster1 = {};
+var monster2 = {};
+var monstersCaught = 0;
+// Handle keyboard controls
+var keysDown = {};
+// Check for keys pressed where key represents the keycode captured
+addEventListener("keydown", function (key) {
+  keysDown[key.keyCode] = true;
+}, false);
+addEventListener("keyup", function (key) {
+  delete keysDown[key.keyCode];
+}, false);
+// Reset the player and monster positions when player catches a monster
+var reset = function () {
+  // Reset player's position to centre of canvas
+  hero.x = canvas.width /2;
+  hero.y = canvas.height /2 ;
+  // Place the monster somewhere on the canvas randomly
+  monster1.x = 32 + (Math.random() * (canvas.width - 64));
+  monster1.y = 32 + (Math.random() * (canvas.height - 64));
+  
+  monster2.x = 32 + (Math.random() * (canvas.width - 64));
+  monster12y = 32 + (Math.random() * (canvas.height - 64));
+
+
+};
+// Update game objects - change player position based on key pressed
+var update = function (modifier) {
+  if (38 in keysDown) { // Player is holding up key
+   new_position =  hero.y - (hero.speed * modifier);
+    if (new_position > 0) {
+        hero.y = new_position;
+    }
+}
+if (40 in keysDown) { // Player is holding down key
+    new_position =  hero.y + (hero.speed * modifier);
+    console.log(new_position)
+    console.log(hero.height)
+    if (new_position < (canvas.height - hero.height)) {
+        hero.y = new_position;
+    }
+}
+if (37 in keysDown) { // Player is holding left key
+    new_position =  hero.x - (hero.speed * modifier);
+    if (new_position > 0) {
+        hero.x = new_position;
+    }
+}
+if (39 in keysDown) { // Player is holding right key
+    new_position =  hero.x + (hero.speed * modifier);
+    console.log(new_position)
+    console.log(hero.width)
+    if (new_position < (canvas.width - hero.width)) {
+        hero.x = new_position;
+    }
+}
+  // Check if player and monster collider
+  if (
+    hero.x <= (monster.x + 32)
+    && monster.x <= (hero.x + 32)
+    && hero.y <= (monster.y + 32)
+    && monster.y <= (hero.y + 32)
+  ) {
+    ++monstersCaught;
+    reset();
+  }
+};
```

Comment:
Copy lines 114 to 124 and paste it twice. Once with all the `monster.x` set to `monster1.x`  and `monster.y` set to `monster1.y` and again with `monster2`